### DISCUSSION TITLE
repetitive sequences in CDS regions

### DISCUSSION THREAD

**Discussion by glopatriello** [Gaius-Augustus/Augustus#319] [Q&A] [answered]
Hi everyone!

I am using Augustus(hints-based)  to perform an ab initio gene prediction in a non model organism. The obtained results reports some genes with repetitive sequences in CDS region. With manual inspection, I have observed that these repetitive regions were softmasked and according to aligned RNA-seq data these regions should be introns instead. I was wondering how I can increase the Augustus weigths to increase the score for gene models that do not report these repetitive sequences,

Best,
Giulia

**Comment by KatharinaHoff** [comment, marked as answer]
Fine tuning parameters is a tedious task... and it may even be
locus-specific.

You could hard mask the sequences that you absolutely don't want to be part
of CDS. That might be an easy fix in your case.
